docs/appendices/release-notes/5.10.12.rst

```diff
@@ -47,5 +47,9 @@ See the :ref:`version_5.10.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an issue that could cause ``SELECT`` statements to get stuck if executed
+  via a PostgreSQL client that set the ``maxRow`` option as part of an
+  ``EXECUTE`` message and it matched the number of result rows exactly.
```

Comment:
Isn't it about any batch execution in general? 

Context: was discussing if this fix helps with stuck INSERT when ingested via flink that uses prepared statements/ batch inserts.